curriculum/challenges/english/blocks/workshop-report-card-printer/694648acde178bb8202d9513.md

```diff
@@ -0,0 +1,42 @@
+---
+id: 694648acde178bb8202d9513
+title: Step 6
+challengeType: 0
+dashedName: step-6
+---
+
+# --description--
+
+The student name should follow the same format as the other details.
+
+Remove the earlier outputs of the `name` variable. Then, print `name` and `type(name)` together on one line like the previous step.
+
+# --hints--
+
+Remove `print()` statements from Step 1 and 2.
```

Comment:
We need to separate tests here:
- You should not have `print(name)` in your code.
- You should not have `print(type(name))` in your code.